package.json

```diff
@@ -21,6 +21,7 @@
     "eventemitter3": "1.1.1",
     "google-protobuf": "3.6.1",
     "hex2dec": "1.0.1",
+    "opentracing": "^0.14.4",
```

Comment:
why was open-tracing moved to be a dependency? I think consumers are expected to import the version of open-tracing they wish to use themselves. but perhaps it should be a peerDependency.

Reply:
It's a mandatory dependency. For example, `src/imp/span_imp.js` has:

```
import * as opentracing from 'opentracing';
...
export default class SpanImp extends opentracing.Span {
```

which means it's required at `require` / `import` time!

Could be a peerDep, I suppose.
```